src/arguments/api_test.py

```diff
@@ -42,7 +42,39 @@ def fetch_input_url(cls):
             "request_url" : request_url,
             "request_headers" : request_headers,
         }
-
+    @classmethod
+    def read_data_from_file(cls):
+        filename = input("Enter a filename (response_data.json)")
+        if filename == '':
+            print("filename empty, so default file (response_data.json) is used ")
```

Comment:
I think you haven't changed this 😑 . Hope you will make the changes.

Reply:
I have already changed this part of code, may be you are looking at ``` save_response_data function ``` that is not related to this issue, if you still want me to change that old function, I would change and update the commits

Reply:
Its ok, thank you 😄